net-irc/hexchat/hexchat-9999.ebuild

```diff
@@ -1,132 +1,88 @@
-# Copyright 1999-2014 Gentoo Foundation
+# Copyright 1999-2016 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=5
+EAPI=6
 
-PYTHON_COMPAT=( python2_7 python3_3 python3_4 )
-inherit autotools eutils fdo-mime gnome2-utils mono-env multilib python-single-r1 git-2
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+inherit fdo-mime gnome2-utils mono-env multilib python-single-r1
 
 DESCRIPTION="Graphical IRC client based on XChat"
-HOMEPAGE="http://hexchat.github.io/"
-SRC_URI=""
-EGIT_REPO_URI="git://github.com/hexchat/hexchat.git"
+HOMEPAGE="https://hexchat.github.io/"
+
+if [[ "${PV}" == "9999" ]] ; then
+	inherit git-r3
+	SRC_URI=""
+	EGIT_REPO_URI="git://github.com/hexchat/hexchat.git"
+else
+	SRC_URI="https://dl.hexchat.net/hexchat/${P}.tar.xz"
+	KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux"
+fi
 
 LICENSE="GPL-2 plugin-fishlim? ( MIT )"
 SLOT="0"
-KEYWORDS=""
-IUSE="dbus +gtk ipv6 libcanberra libnotify libproxy libressl nls ntlm perl +plugins plugin-checksum plugin-doat plugin-fishlim plugin-sysinfo python spell ssl theme-manager"
-REQUIRED_USE="plugins? ( python? ( ${PYTHON_REQUIRED_USE} ) )"
+IUSE="dbus debug +gtk libcanberra libnotify libproxy libressl lua nls perl plugin-checksum plugin-doat plugin-fishlim plugin-sysinfo python spell ssl theme-manager"
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
 
-DEPEND="dev-libs/glib:2
-	dbus? ( >=dev-libs/dbus-glib-0.98 )
+COMMON_DEPEND="dev-libs/glib:2
+	dbus? ( sys-apps/dbus )
 	gtk? ( x11-libs/gtk+:2 )
 	libcanberra? ( media-libs/libcanberra )
 	libproxy? ( net-libs/libproxy )
 	libnotify? ( x11-libs/libnotify )
+	lua? ( dev-lang/lua:= )
 	nls? ( virtual/libintl )
-	ntlm? ( net-libs/libntlm )
-	plugins? (
-		perl? ( >=dev-lang/perl-5.8.0 )
-		plugin-sysinfo? ( sys-apps/pciutils )
-		python? ( ${PYTHON_DEPS} )
-	)
+	perl? ( dev-lang/perl )
+	plugin-sysinfo? ( sys-apps/pciutils )
+	python? ( ${PYTHON_DEPS} )
 	spell? ( app-text/iso-codes )
 	ssl? (
-		!libressl? ( dev-libs/openssl:0 )
-		libressl? ( dev-libs/libressl )
+		!libressl? ( dev-libs/openssl:0= )
+		libressl? ( dev-libs/libressl:0= )
 	)
 	theme-manager? ( dev-lang/mono )"
-RDEPEND="${DEPEND}
+RDEPEND="${COMMON_DEPEND}
 	spell? ( app-text/enchant )"
-DEPEND="${DEPEND}
+DEPEND="${COMMON_DEPEND}
 	app-arch/xz-utils
 	virtual/pkgconfig
-	nls? ( dev-util/intltool )
+	dev-util/intltool
 	theme-manager? ( dev-util/monodevelop )"
 
+src_prepare() {
+	if [[ ${PV} == "9999" ]]; then
+		./autogen.sh || die
```

Comment:
you sure you don't need `autotools` anymore here? Consider readding it to both ebuilds, in order to keep them in sync.